index.html

```diff
@@ -0,0 +1,161 @@
+                <audio src="./profile-pics/[YT2mp3.info] - The Shire.mp4" controls autoplay></audio>
+            </header>
+            <div class="text g">
+                <div class="prof-border">
+                    <img src="./profile-pics/Gimli.jpg" alt="Gimli" class="profile-pic" >
+                </div>
+                <p>What's happening?</p>
+                <div></div>
+                <div class="text-grid g">
+                    <img src="./twitter-svg/image-photography-icon.svg" alt="Picture icon" class="logoz">
+                    <img src="./twitter-svg/gif-icon.svg" alt="Gif icon" class="logoz">
+                    <img src="./twitter-svg/bar-chart-icon.svg" alt="Bar chart icon" class="logoz">
+                    <img src="./twitter-svg/smiley-icon.svg" alt="Emoji icon" class="logoz">
+                    <img src="./twitter-svg/information-date-calendar-icon.svg" alt="Calender icon" class="logoz">
+                    <button>Tweet</button>
+                </div>
+            </div>
+            <div class="shadow-gap g"></div>
+            <div class="content g">
+                <div  class="each-content g">
+                    <div>
+                        <div class="g" id="refresh">
+                            <img src="./twitter-svg/rotate-left-arrow-icon.svg" alt="Refresh icon" class="logo-content">
+                        </div>
+                        <div class="prof-border">
+                            <img src="./profile-pics/aragorn.webp" alt="Aragorn" class="profile-pic aragorn">
+                        </div>
+                    </div>
+                    <div>
+                        <span><b>Carry_Artist_Legolas Retweeted</b></span>
+                        <div class="g elipses-grid">
+                            <p><b>Orc Slayers </b><span>@KingOfMiddleDerth . 36m</span></p>
+                            <img src="./twitter-svg/3-horizontal-dots-icon.svg" alt="More icon" class="elipses">
+                        </div>
+                        <p>We are going on a mission to slay some more of the Ops &#128526;</p>
+                        <p><span>@Carry_Artist_Legolas </span> Lookin in form, you wanna join us?</p>
+                        <div class="g logo-grid">
+                            <span><img src="./twitter-svg/chat-message-icon.svg" alt="Chat bubble" class="logo-content"> 6</span>
+                            <span><img src="./twitter-svg/rotate-left-arrow-icon.svg" alt="Refresh icon" class="logo-content"> 13</span>
+                            <span><img src="./twitter-svg/like-heart-round-icon.svg" alt="Like heart icon" class="logo-content"> 25</span>
+                            <span><img src="./twitter-svg/upload-outbox-line-icon.svg" alt="Upload icon" class="logo-content"> 2</span>
+                            <span><img src="./twitter-svg/pyramid-icon.svg" alt="Tip icon" class="logo-content"> 1</span>
+                        </div>
+                    </div>
```

Comment:
Unsure why you have used a <span>? You've also got a class for each image

Reply:
sorry bro, i didn't quite understand which part you mean, what do you mean by using a '?'  ?

Reply:
My bad xD there is <span> tags used and also classes used for the same sections. I'm unsure why both have been used in this instance

Reply:
span tags. it deletes that section of my text when I contain it

Reply:
I just used span to have the text right next to the image, so it would be in 1 container, also used span because it had some css decorations of how i wanted the numbers to be already